### ISSUE TITLE
[Resolved] Variational autoencoder example fails

### ISSUE THREAD

**Issue by JamesBrofos** [fchollet/keras#3232]
Never mind: I see there was a recent pull request that resolved this issue.

I am using Keras version 1.0.5 with Python3. Using the variational autoencoder example found here: [https://raw.githubusercontent.com/fchollet/keras/master/examples/variational_autoencoder.py](https://raw.githubusercontent.com/fchollet/keras/master/examples/variational_autoencoder.py)

I obtain the error:

`Exception: The `compute_mask` method of layer "lambda_1" should return one mask tensor per output tensor of the layer. Found: [None, None]`

I am sure there is an easy fix for this error, but it would be good to out out-of-the-box functioning examples. I've ensured that this behavior is the same across both the Theano and Tensorflow backends.

**Comment by adelsalehali1982** [comment]
I still have the same problem what is the solution?
